app/src/main/java/com/xa/xpensauditor/EmailSender.java

```diff
@@ -16,20 +16,36 @@
 import org.json.JSONException;
 import org.json.JSONObject;
 
+/**
+ * This class is a async task to send emails using the MailJet API. This does
+ * not block the main thread and can be used from the UI. The API is rate-limited to
+ * 500 emails per day.
+ */
 public class EmailSender extends AsyncTask<Void, Integer, Boolean> {
 
     private String email;
     private String amt;
     private String shop;
     private String cat;
 
+    /**
```

Comment:
⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.javadoc.SummaryJavadocCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Summary javadoc is missing.